platforms/r3-corda/configuration/samples/network-minikube.cordav2.yaml

```diff
@@ -0,0 +1,449 @@
+      k8s:
+        region: "cluster_region"
+        context: "minikube"
+        config_file: "/home/blockchain-automation-framework/build/config"
+
+      # Hashicorp Vault server address and root-token. Vault should be unsealed.
+      # Do not check-in root_token
+      vault:
+        url: "vault_addr"
+        root_token: "vault_root_token"
+
+      # Git Repo details which will be used by GitOps/Flux.
+      # Do not check-in git_password
+      gitops:
+        git_ssh: "gitops_ssh_url"         # Gitops ssh url for flux value files like "ssh://git@github.com/hyperledger-labs/blockchain-automation-framework.git"
+        branch: "gitops_branch"           # Git branch where release is being made
+        release_dir: "gitops_release_dir" # Relative Path in the Git repo for flux sync per environment. 
+        chart_source: "gitops_charts"     # Relative Path where the Helm charts are stored in Git repo
+        git_push_url: "gitops_push_url"   # Gitops https URL for git push like "github.com/hyperledger-labs/blockchain-automation-framework.git"
+        username: "git_username"          # Git Service user who has rights to check-in in all branches
+        password: "git_password"          # Git Server user password
+        email: "git_email"                # Email to use in git config
+        private_key: "path_to_private_key"          # Path to private key file which has write-access to the git repo
+      
+      # Cordapps Repository details (optional if cordapps jar are store in a repository)
+      cordapps:
+        jars: 
+        - jar:
+            # e.g https://alm.accenture.com/nexus/repository/AccentureBlockchainFulcrum_Release/com/supplychain/bcc/cordapp-supply-chain/0.1/cordapp-supply-chain-0.1.jar
+            url: 
+        - jar:
+            # e.g https://alm.accenture.com/nexus/repository/AccentureBlockchainFulcrum_Release/com/supplychain/bcc/cordapp-contracts-states/0.1/cordapp-contracts-states-0.1.jar
+            url: 
+        username: "repository_username"
+        password: "repository_password"
+      
+      services:
+        peers:
+        - peer:
+          name: carrier
+          subject: "O=Carrier,OU=Carrier,L=51.50/-0.13/London,C=GB" # This is the node subject. L=lat/long is mandatory for supplychain sample app
+          type: node
+          p2p:
+            port: 10002
+            targetPort: 10002
+            ambassador: 10030       #Port for ambassador service (use one port per org if using single cluster)
+          rpc:
+            port: 10003
+            targetPort: 10003
+          rpcadmin:
+            port: 10005
+            targetPort: 10005
+          dbtcp:
+            port: 9101
+            targetPort: 1521
+          dbweb:             
+            port: 8080
+            targetPort: 81
+          springboot:
+            targetPort: 20001
+            port: 20001 
+          expressapi:
+            targetPort: 3000
+            port: 3000
+
+    - organization:
```

Comment:
no need to include store and warehouse orgs. this has to be minimal. 3 orgs are enough.